app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt-based Markdown editor"
+DESCRIPTION="CuteMarkEd is a Qt-based Markdown editor with live HTML preview, \
++math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose
+	2007 David Loren Parsons
+	2011-2012 Ali Rantakari
+	2012 CodeCatalyst, LLC"
+LICENSE="GNU GPL v2
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="78a41808c2f0452375810abdff76eeaaee012f8d1368a2b8772ec6b4d2ceeec8"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cutemarked$secondaryArchSuffix = $portVersion
+	app:CuteMarkEd$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libhunspell_1.6$secondaryArchSuffix
+	lib:libmarkdown$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libhunspell_1.6$secondaryArchSuffix
+	devel:libmarkdown$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5QmlDebug$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	qmake CuteMarkEd.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install	-d $appsDir
```

Comment:
Oops, there is a tab (instead of a space) between the "install" and the "-d $appsDir".
Could you please replace that tab by a white space? Thanks!
(I know it won't make any difference. It's just that we prefer to have tabs only where we can "see" them with the resulting indendation.)

While at it, could you please also sort the `cmd:*` lines in `BUILD_PREREQUIRES`?